### DISCUSSION TITLE
Build Time Error

### DISCUSSION THREAD

**Discussion by zerofront** [vercel/next.js#14360] [Help]
Automatically optimizing pages...

![image](https://user-images.githubusercontent.com/19327366/85128390-bbc55680-b263-11ea-82c0-b29d40075c36.png)

I see the log has been output ”Compiled successfully“

But fail in ssg? 

I dont' need ssg and I use getServerSideProps to fetch data at every request ?

How can I fix this problem ?

**Comment by timneutkens** [comment]
Can you provide a complete reproduction?
